sys-fs/cloudfuse/cloudfuse-1.0.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+inherit eutils
```

Comment:
I prefer putting newlines before and after the inherit lines.  Just personal preference and style not a requirement.  Your choice on whether you like that style.